cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -22,18 +22,18 @@ import (
 	"net"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
keep original better, and others place

Reply:
I'm not sure how this happened. nice catch.